lib/rouge/lexers/sml.rb

```diff
@@ -41,9 +41,7 @@ def self.analyze_text(text)
         rule /[(\[{]/, Punctuation, :main
         rule /[)\]}]/, Punctuation, :pop!
 
-        rule /\b(let|if|local)\b(?!')/, Keyword::Reserved do
-          push; push
```

Comment:
I'm not sure why it pushes two copies of itself. You can try looking at the result if you remove one by running `rackup` and visiting `localhost:9292/sml` in your browser.